mitmproxy/console/flowlist.py

```diff
@@ -18,8 +18,8 @@ def _mkhelp():
         ("d", "delete flow"),
         ("D", "duplicate flow"),
         ("e", "toggle eventlog"),
+        ("f", "set filter view pattern"),
```

Comment:
If I recall the discussion about this correctly, this should just be "filter view".